solr/core/src/java/org/apache/solr/handler/admin/IndexSizeEstimator.java

```diff
@@ -384,16 +386,17 @@ private void estimateTermVectors(Map<String, Object> result) throws IOException
     for (LeafReaderContext leafReaderContext : reader.leaves()) {
       LeafReader leafReader = leafReaderContext.reader();
       Bits liveDocs = leafReader.getLiveDocs();
+      TermVectors termVectors = leafReader.termVectors();
       for (int docId = 0; docId < leafReader.maxDoc(); docId += samplingStep) {
         if (liveDocs != null && !liveDocs.get(docId)) {
           continue;
         }
-        Fields termVectors = leafReader.getTermVectors(docId);
-        if (termVectors == null) {
+        Fields vectors = termVectors.get(docId);
+        if (vectors == null) {
```

Comment:
Why change the variable name?  "vectors" is bringing a more ambiguous word in recent times ("vector search")

Reply:
> Why change the variable name? "vectors" is becoming a more ambiguous word in recent times ("vector search")

I guess it allows the `termVectors` name to be used above where it matches the type and retrieval method name.

How about `fields` as the changed variable name instead of `vectors`?

Reply:
https://github.com/apache/solr/pull/1557/commits/384035bbd589b4cc6f2ec8cc3c076e5c8c7cda3d for `fields` instead of `vectors` name.